app/js/components/demographics.jsx

```diff
@@ -198,187 +209,205 @@ export default class Demographics extends React.Component {
-                        </div>
-                    }
-
-                    <div className='form-group'>
-                        <label className='control-label col-sm-2'> Deceased: </label>
-                        <div className='col-sm-2'>
-                            <input
-                                name='deceased'
-                                type='checkbox'
-                                checked={this.state.deceased}
-                                onChange={this.onDeceased}
-                            />
+                        <div className='form-group demo'>
+                            {this.state.gender === 'M' ?
+                                <label className='control-label col-sm-3'> Male </label>
+                                : <label className='control-label col-sm-3'> Female </label>
+                            }
                         </div>
-                    </div>
+                        <a onClick={this.handleChange}> Edit </a>
+                        <Modal show={this.state.showModal} toggle={this.toggle} className={this.props.className}>
+                            <ModalHeader toggle={this.toggle}>Edit Demographics</ModalHeader>
+                            <ModalBody>
+                                <form className='form-horizontal' onSubmit={this.handleSave}>
+                                    <div className='form-group'>
+                                        <label className='control-label col-sm-2'>Gender:</label>
+                                        <div className='col-sm-5'>
+                                            <label className='radio-inline'>
+                                                <input type='radio' id='inlineRadio1'
+                                                    value='option1' name='male'
+                                                    type='radio'
+                                                    value='M'
+                                                    checked={this.state.gender === 'M'}
+                                                    onChange={this.setGender}
+                                                /> Male
+                            </label>
+                                            <label className='radio-inline'>
+                                                <input type='radio' id='inlineRadio2'
+                                                    value='option2'
+                                                    name='female'
+                                                    type='radio'
+                                                    value='F'
+                                                    checked={this.state.gender === 'F'}
+                                                    onChange={this.setGender}
+                                                /> Female
+                            </label>
+                                        </div>
+                                    </div>
```

Comment:
There are 2  closing label tags  out of indent

Reply:
Corrected. Thanks